conda_smithy/feedstock_io.py

```diff
@@ -75,10 +75,14 @@ def remove_file(filename):
 
 
 def copy_file(src, dst):
-    with io.open(src, "r", encoding="utf-8") as fh_src:
-        with io.open(dst, "w", encoding="utf-8", newline="\n") as fh_dst:
-            for line in fh_src:
-                fh_dst.write(line)
+    try:
+        with io.open(src, "r", encoding="utf-8") as fh_src:
+            with io.open(dst, "w", encoding="utf-8", newline="\n") as fh_dst:
+                for line in fh_src:
+                    fh_dst.write(line)
+    except UnicodeDecodeError:
+        # Leave any other files alone.
+        shutil.copy(src, dst)
```

Comment:
If the line issue exist with text files that fails when opened with `utf-8` that will still exist in this scenario.
But it is better than the current situation and such a scenario would be quite improbable.